privacyidea/lib/tokens/webauthntoken.py

```diff
@@ -901,7 +901,7 @@ def update(self, param, reset_failcount=True):
                     uv_required=uv_req == USER_VERIFICATION_LEVEL.REQUIRED
                 ).verify([
                     # TODO: this might get slow when a lot of webauthn tokens are registered
-                    token.decrypt_otpkey() for token in get_tokens(tokentype=self.type)
+                    token.decrypt_otpkey() for token in get_tokens(tokentype=self.type) if token.get_serial() != self.get_serial()
```

Comment:
I added this because i ran into an issue with the tests (where one webauthn token counted as two because it was also checked here).
This specific part is described in step 22 here: https://www.w3.org/TR/webauthn/#sctn-registering-a-new-credential
There should be a check whether the credentialID (saved as the otpkey) is already used by a different token.
We could reduce this to "assigned" token since all others aren't functional.